lib/src/models/retry_event_model.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:async';
+
+/// Retries the `retryFunction` after a set `timeInterval` until `dispose` is called
+class RetryEventModel {
```

Comment:
Does that need to be public? Also it has a very enterprisy name.

Reply:
Isn't exported though? (also I would say is a useful class, just not something the matrix package should provide I guess)

Reply:
_RetryEventModel would be private to the file not the library, not exporting is what you want here

Reply:
I guess, although you could also just put it in the file you use it in for now. But yeah, I guess that is fine.